devel-requirements.in

```diff
@@ -6,7 +6,6 @@ djangorestframework-stubs
 factory-boy
 freezegun
 k5test
-lxml
```

Comment:
Just an idea, we do have a `testenv:mypy` in our `tox.ini` and according to [this](https://github.com/pre-commit/mirrors-mypy/issues/30), the `lxml` is probably needed to run mypy with the `--html-report` parameter. :thinking:

Reply:
Good catch, I will just update the lib in that case